### PR TITLE
Create 0x555d051538c7a13712f1f590fa6b4c176ca4529f.json

### PR DESCRIPTION
**Pull request by iOWNToken**
iOWN Token (https://www.iowntoken.com)

### CODE REVIEW

File: src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "symbol": "iOWN",
+  "name": "iOWN Token",
+  "type": "ERC20",
+  "address": "0x555d051538c7a13712f1f590fa6b4c176ca4529f",
+  "ens_address": "",https://www.youtube.com/channel/UC3gsYQsL0yYC_ShOYUGP5WA
```
Comment (gamalielhere) [inline src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json]: Please remove the youtube link

Comment (axelsayegh) [reply src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json]: @gamalielhere hi, can you please check if we can get listed now?

**Comment by axelsayegh** [comment]
Hi @gamalielhere, any issues or we're ok to be added?

File: src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json (hunk 1 above)
Comment (gesseekur) [reply src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json]: yup , will merge and add

Comment (axelsayegh) [reply src/tokens/eth/0x555d051538c7a13712f1f590fa6b4c176ca4529f.json]: Thanks @gesseekur

**Pull request merged**